src/app/shared/libraries/accordion/accordion.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
same here. Prove your component works. what if I delete this <ng-content></ng-content>. the test will still pass and that is not what we want